.env.example

```diff
@@ -1,4 +1,4 @@
-REACT_APP_FARM_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/crisog/alpha-rinkeby-subgraph
+REACT_APP_FARM_SUBGRAPH_URL="https://thegraph.com/explorer/subgraph/evalir/regen-farm-rinkeby"
```

Comment:
We can probably remove this and leave Cris's subgraph :).

Reply:
Fixed in df9d53e256841982537dfa54187494c16724bd9b